docker/package.json

```diff
@@ -6,5 +6,8 @@
   "license": "ISC",
   "dependencies": {
     "devcert": "~1.1.0"
+  },
+  "resolutions": {
+    "minimist": "^1.2.2"
```

Comment:
Using `resolutions` effectively "steps over" any transitive dependency requirements on older versions. This may cause issues, especially in this case where the resolution has a major bump.

The only other alternative to fixing this would be to update down the chain of transitive deps, but each of those packages also needs to get fixed/published. So this is the fastest way to resolve the security issues but it may cause a bug/break in the app.